src/action/macos/mod.rs

```diff
@@ -4,8 +4,9 @@
 pub(crate) mod bootstrap_launchctl_service;
 pub(crate) mod configure_remote_building;
 pub(crate) mod create_apfs_volume;
-#[cfg(not(feature = "nix-community"))]
-pub(crate) mod create_enterprise_edition_volume;
+// #[cfg(not(feature = "nix-community"))]
```

Comment:
```suggestion
#[cfg(not(feature = "nix-community"))]
```

Reply:
We backed off of this approach to avoid having to go weed out and special/case the uses (but I should have cleaned up the commented-out copies, so it's right to poke at them).